### ISSUE TITLE
HSAF SMDAS SWI

### ISSUE THREAD

**Issue by wpreimes** [TUW-GEO/ascat#43]
Do you want to include the [SMDAS ](http://hsaf.meteoam.it/description-sm-das-2.php)data reader in this package? https://github.com/wpreimes/ascat/blob/smdas_reader/src/ascat/smdas.py

Then I'll fix some issues, clean up, write tests etc for it.

The data itself is a bit awkward, corrupt files, missing variables, deviating time stamps in metadata and filename etc.
Therefore the reader is also a bit messy...

**Comment by sebhahn** [comment]
Which products do you mean by SMDAS exactly?

**Comment by wpreimes** [comment]
H-SAF SM-DAS-2 H-14 root zone soil moisture index product, based on ASCAT surface soil moisture data assimilation in the ECMWF Land Data Assimilation System

**Comment by sebhahn** [comment]
Yes it makes sense to add a reader, but sooner or later it this should be moved to its own package here https://github.com/H-SAF

**Comment by sebhahn** [comment]
Same for all other H SAF SM products

**Comment by wpreimes** [comment]
If you want I can make a new package for the SMDAS data now already. Just tell me what you prefer.

**Comment by sebhahn** [comment]
Probably it makes sense to start with a new package, but I wouldn't call it SMDAS.

However, first I need to think about how to structure the packages in H SAF (package per product, package per variable, ...). What would you prefer as a user?

**Comment by sebhahn** [comment]
@davidfairbairn do you have an opinion how to structure the python packages on H SAF?

**Comment by wpreimes** [comment]
From a user perspective it would be nice to have everything in one place. But from a developer point of view I think a package per product makes more sense than a package per variable, because of different file types, dependencies, resolutions etc. So maybe multiple smaller readers and one toolbox to collect them?

**Comment by sebhahn** [comment]
I like the idea with the toolbox, but typically products are quite similar in their format, type, etc. so probably a package per variable makes more sense? surface soil moisture and root zone soil moisture separated
